datafusion/proto/src/logical_plan/to_proto.rs

```diff
@@ -218,7 +218,7 @@ impl TryFrom<&DataType> for protobuf::arrow_type::ArrowTypeEnum {
             DataType::Decimal256(_, _) => {
                 return Err(Error::General("Proto serialization error: The Decimal256 data type is not yet supported".to_owned()))
             }
-            DataType::Map(_, _) => {
+            DataType::Map(_, _) | DataType::RunEndEncoded(_, _) => {
```

Comment:
I recommend either updating the error message here or adding a separate clause for RunEndEncoded